gui/default/index.html

```diff
@@ -122,6 +122,39 @@ <h3 class="panel-title">
       </div>
     </div>
 
+    <!-- Panel: Closed, no auth -->
+
+    <div ng-if="closedNoAuth" class="row">
+      <div class="col-md-12">
+        <div class="panel panel-warning">
+          <div class="panel-heading">
+            <h3 class="panel-title">
+              <div class="panel-icon">
+                <span class="fas fa-exclamation-circle"></span>
+              </div>
+              <span translate>Warning!</span>
+            </h3>
+          </div>
+          <div class="panel-body">
+            <p>
+            <span translate>If this is a shared computer, other users might be able to access the admin interface.</span>
+            <span translate>Please set a GUI Authentication User and Password in the Settings dialog.</span>
```

Comment:
I wouldn't capitalize user, password and settings. And I'd word it something like "You can set a user and password in the GUI tab of the settings dialog to restrict access".